pubspec.yaml

```diff
@@ -3,12 +3,12 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: ffigen
-version: 4.1.2
+version: 5.0.0-dev.1
```

Comment:
nit: I think we can start counting with dev.0

Reply:
Ofcourse 🙈 and Happy New year 🎉